indico/htdocs/js/indico/modules/contributions/common.js

```diff
@@ -236,6 +236,50 @@
             var target = $(evt.target);
             reloadManagementAttachmentInfoColumn(target.data('locator'), target.closest('td'));
         });
+
+
+        $('#subcontribution-list table').sortable({
+            items: '.js-sortable-subcontribution-row',
+            handle: '.ui-sortable-handle',
+            placeholder: 'sortable-placeholder',
+            tolerance: 'pointer',
+            distance: 10,
+            axis: 'y',
+            containment: '#subcontribution-list table',
+            start: function(e, ui){
+                ui.placeholder.height(ui.helper.outerHeight());
+            },
+            update: function(e, ui) {
+                var self = $(this);
+                var oldIndex = ui.item.data('position');
```

Comment:
isn't there a sortable callback that's only invoked when something actually changed? because in that case you could probably get rid of data-position etc